python/pyarrow/tests/test_compute.py

```diff
@@ -137,7 +137,7 @@ def test_option_class_equality():
                 pytest.fail(f"Options class is not tested: {cls}")
     for option in options:
         assert option == option
-        assert repr(option)
+        assert repr(option).startswith(option.__class__.__name__)
```

Comment:
Can you test the actual repr for one or two classes somewhere?

Reply:
I added some assertions at the bottom of the test.